slices/busybox.yaml

```diff
@@ -0,0 +1,8 @@
+package: busybox
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
```

Comment:
Why? :thinking: 

```
# ldd /usr/bin/busybox
	not a dynamic executable
```

Reply:
That would probably need digging around the source code, cause:
```bash
$ apt info busybox | grep Depends

Depends: libc6 (>= 2.34)
```

Reply:
Needs it for compiling, but not for runtime. Thanks @woky !!!! =)

Reply:
Oh, loader, you are right. Definitely a good idea to make a loader only libc slice.